server/chat-commands/core.ts

```diff
@@ -1275,15 +1310,15 @@ exports.commands = {
 			return;
 		}
 		if (!target) return this.errorReply("Provide a valid format.");
-		let originalFormat = Dex.getFormat(target);
+		const originalFormat = Dex.getFormat(target);
 		// Note: The default here of [Gen 7] Pokebank Anything Goes isn't normally hit; since the web client will send a default format
-		let format = originalFormat.effectType === 'Format' ? originalFormat : Dex.getFormat('[Gen 7] Pokebank Anything Goes');
-		if (format.effectType !== 'Format') return this.popupReply("Please provide a valid format.");
+		const tier = originalFormat.effectType === 'Format' ? originalFormat : Dex.getFormat('[Gen 7] Pokebank Anything Goes');
```

Comment:
```suggestion
		const tier = originalFormat.effectType === 'Format' ? originalFormat : Dex.getFormat('[Gen 8] Anything Goes');
```

gen 7 hasnt existed for years